pkg/services/sqlstore/migrations/object_store_mig.go

```diff
@@ -8,36 +8,47 @@ import (
 	"github.com/grafana/grafana/pkg/setting"
 )
 
+func getKeyColumn(name string, isPrimaryKey bool) *migrator.Column {
+	return &migrator.Column{
+		Name:         name,
+		Type:         migrator.DB_NVarchar,
+		Length:       1024,
+		Nullable:     false,
+		IsPrimaryKey: isPrimaryKey,
+		IsLatin:      true, // only used in MySQL
+	}
+}
+
 func addObjectStorageMigrations(mg *migrator.Migrator) {
 	tables := []migrator.Table{}
 	tables = append(tables, migrator.Table{
 		Name: "object",
 		Columns: []*migrator.Column{
 			// Object key contains everything required to make it unique across all instances
-			// orgId+scope+kind+uid
-			{Name: "key", Type: migrator.DB_NVarchar, Length: 1024, Nullable: false, IsPrimaryKey: true},
+			// orgId + scope + kind + uid
+			getKeyColumn("key", true),
 
 			// This is an optimization for listing everything at the same level in the object store
-			{Name: "parent_folder_key", Type: migrator.DB_NVarchar, Length: 1024, Nullable: false},
+			getKeyColumn("parent_folder_key", false),
 
 			// The object type
 			{Name: "kind", Type: migrator.DB_NVarchar, Length: 255, Nullable: false},
 
 			// The raw object body (any byte array)
-			{Name: "body", Type: migrator.DB_Blob, Nullable: false},
+			{Name: "body", Type: migrator.DB_LongBlob, Nullable: false},
 			{Name: "size", Type: migrator.DB_BigInt, Nullable: false},
-			{Name: "etag", Type: migrator.DB_NVarchar, Length: 32, Nullable: false}, // md5(body)
+			{Name: "etag", Type: migrator.DB_NVarchar, Length: 32, Nullable: false, IsLatin: true}, // md5(body)
 			{Name: "version", Type: migrator.DB_NVarchar, Length: 128, Nullable: false},
 
 			// Who changed what when -- We should avoid JOINs with other tables in the database
-			{Name: "updated", Type: migrator.DB_DateTime, Nullable: false},
-			{Name: "created", Type: migrator.DB_DateTime, Nullable: false},
+			{Name: "updated_at", Type: migrator.DB_BigInt, Nullable: false},
```

Comment:
switched from Date to int64 since that is what our protobuf will send -- also date support across the three engines has awkward differences